README.md

```diff
@@ -506,6 +506,7 @@ On Windows using Visual Studio
         * Note: Java development kit >= 9 requires CMake >= 3.10.
     * **Python scripting** (optional): Python 2 >= 2.7 or Python 3 >= 3.5
         * [Anaconda](https://store.continuum.io/cshop/anaconda/)
+        * Must provide the NumPy package; this should come with Anaconda.
```

Comment:
Noticed that the Anaconda url above this link gives a 404. This is the correct link: https://www.anaconda.com/distribution/

Reply:
Thank you; fixed.